### PR TITLE
new devices

### PR DESCRIPTION
**Pull request by erelke**
A couple of new devices. There are already similar ones, but the devices don't work with them.

### CODE REVIEW

**Comment by make-all** [comment]
One of the commits says "hungarian translation".  Did you make a modification to the Hungarian translations which you forgot to stage when committing?

**Comment by make-all** [comment]
I also suggest breaking these into separate pull requests.  Some can probably be merged quite quickly, others may have some work to do, or even be better dealt with by modifying existing configs. Submitting all together like this delays them all until the most time consuming one has finally been dealt with.

File: custom_components/tuya_local/devices/earu_smart_wifi_circuit_breaker_with_energy_monitor.yaml [hunk 1]
```diff
@@ -0,0 +1,364 @@
+products:
+  - id: ine4hhapkgxonij1
+    name: EARU smart breaker
+    # https://www.aliexpress.com/item/1005004894566311.html
+
```
Comment (make-all) [inline custom_components/tuya_local/devices/earu_smart_wifi_circuit_breaker_with_energy_monitor.yaml:5]: The config still needs a top level name, I was suggesting a generic name like "Smart breaker" for the top level name, and move the brand and model number to the products name.

But I also noted that this config does not appear to be any different than `smartmcb_smt006_energymeterv2.yaml` so probably a better solution is to add the product id and name to that config, and delete this one.

File: custom_components/tuya_local/devices/switch_module_rdcbc_t_j_z_j.yaml [hunk 2]
```diff
@@ -0,0 +1,150 @@
+name: Energy monitoring RDCBC
```
Comment (make-all) [inline custom_components/tuya_local/devices/switch_module_rdcbc_t_j_z_j.yaml:1]: This config also seems like it should match one of the generic smartplug configs already.

File: custom_components/tuya_local/devices/earu_smart_wifi_circuit_breaker_with_energy_monitor.yaml [hunk 3]
```diff
@@ -0,0 +1,366 @@
...
+        type: integer
+        readonly: true
+        persist: true
+        unit: kWh
+        class: total_increasing
+        name: sensor
+        mapping:
+          - scale: 100
+  - entity: sensor
+    name: 'Breaker ID'
```
[the remark this answers is not in the document]
Comment (erelke) [reply custom_components/tuya_local/devices/earu_smart_wifi_circuit_breaker_with_energy_monitor.yaml:36]: How can add as attribute?

File: custom_components/tuya_local/devices/earu_smart_wifi_circuit_breaker_with_energy_monitor.yaml (hunk 1 above)
Comment (erelke) [reply custom_components/tuya_local/devices/earu_smart_wifi_circuit_breaker_with_energy_monitor.yaml:6]: the device does not work with the smartmcb_smt006_energymeterv2.yaml config

File: custom_components/tuya_local/devices/earu_smart_wifi_circuit_breaker_with_energy_monitor.yaml (hunk 3 above)
[the remark this answers is not in the document]
Comment (make-all) [reply custom_components/tuya_local/devices/earu_smart_wifi_circuit_breaker_with_energy_monitor.yaml:36]: Actually it already is `breaker_number` above

File: custom_components/tuya_local/devices/earu_smart_wifi_circuit_breaker_with_energy_monitor.yaml (hunk 1 above)
Comment (make-all) [reply custom_components/tuya_local/devices/earu_smart_wifi_circuit_breaker_with_energy_monitor.yaml:6]: Based on the dps mappings, it should, so you'll have to be more specific what you mean by "does not work".

File: custom_components/tuya_local/devices/simple_contact_sensor.yaml [hunk 4]
```diff
@@ -0,0 +1,18 @@
+name: Simple (door/window) sensor
+products:
+  - id: 0mjj67aaztcfmc3i
+    name: AW201_XR3
+primary_entity:
+  entity: cover
```
Comment (make-all) [inline custom_components/tuya_local/devices/simple_contact_sensor.yaml:6]: If it is just a sensor, then it should be implemented as a binary_sensor.
cover is for control of curtains/doors etc.

**Comment by make-all** [comment]
Manually merged to resolve conflicts and exclude duplicate config with generic smartplugv2_energyv2

**Pull request closed without merging**